src/Arcus.Templates.AzureFunctions.EventHubs/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM mcr.microsoft.com/azure-functions/dotnet:4 AS base
 WORKDIR /app
 EXPOSE 80
 
-FROM mcr.microsoft.com/dotnet/sdk:7.0.203-bullseye-slim AS build
+FROM mcr.microsoft.com/dotnet/sdk:8.0-bookworm-slim AS build
```

Comment:
Not sure if these are correct, seems like the most simple ones. I like simple 😅.

Reply:
I find those names always confusing, but it might be me :)

Reply:
Bookworm seems to be debian (12).
Bullseye is also debian (11), so that's ok.